app/views/articles/show.html.erb

```diff
@@ -14,7 +14,7 @@
           clearInterval(waitingOnPostscribe);
           var els = document.getElementsByClassName("ltag_gist-liquid-tag");
           for (var i = 0; i < els.length; i++) {
-            var current = els[i];
+            let current = els[i];
```

Comment:
This looks like it should fix the issue. If you don't have time to get all set up on your local machine, you can use GitPod. See our [documentation on Gitpod](https://docs.dev.to/installation/gitpod/).